.azure-pipelines/test_plan.py

```diff
@@ -466,6 +482,66 @@ def poll(self, test_plan_id, interval=60, timeout=-1, expected_state=""):
         required=False,
         help="Repository name from Azure Pipelines"
     )
+    parser_create.add_argument(
+        "--testbed-name",
+        type=str,
+        dest="testbed_name",
+        nargs='?',
+        const='',
```

Comment:
Why need to specify `const`?

Reply:
https://docs.python.org/3/library/argparse.html#nargs

We combine nargs='?' and const='', to support the scenario that the parameter is specified but no value is followed, like
--foo-0 a --testbed_name --foo-1 b
This is for us can reuse the script with different parameters everywhere.